### ISSUE TITLE
Have to connect to QGC first and then run sample app, otherwise telemetry won't display.

### ISSUE THREAD

**Issue by mmanionRIIS** [mavlink/MAVSDK-Swift#214]
When I run the MavSDK sample app, I am able to connect to my mavlink drone. But I’m not able to read much of the telemetry. However, if I connect to QGroundControl first, then close out of QGC and then run the sample app, it does pick up the Telemetry.   We are using "udp://:14550" instead of the default "udp://:14540" if that makes a difference.

Any ideas on what may be causing this? 

_Without connecting to QGC first:_

![IMG_0093](https://user-images.githubusercontent.com/100305005/183743493-9e6f50ff-ad44-4404-ba1f-55957488f11b.PNG)

_With connecting to QGC first:_

![IMG_0098](https://user-images.githubusercontent.com/100305005/183743497-dd888488-b7db-428c-9dfc-46161778fc8c.PNG)

**Comment by julianoes** [comment]
What telemetry does the drone send by default? And is it a PX4 or ArduPilot drone?

**Comment by mmanionRIIS** [comment]
It is Ardupilot. I'm not sure what you mean by "send by default". 

The 1st screenshot above shows what happens if you connect without connecting to QGC first, but I'm not sure if that is what you're looking for.

**Comment by mmanionRIIS** [comment]
I tried implementing "setRate" functions for GPS, Battery, and Position. I ran it with the setRate in the TelemetryViewModel and in the TelemetryDetailViewModel, adding it into the ObserveTelemetry functions in both cases. No luck. 

I wasn't sure if I was setting the right Hertz so I tried at 1,5,10,100 all different levels. 

<img width="511" alt="Screen Shot 2022-08-11 at 2 24 53 PM" src="https://user-images.githubusercontent.com/100305005/184212314-272e87e7-d80d-46e7-be3d-33d4507bbd79.png">
"
